site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+
+| Property                                    | Description                                                                                                                                                                                                                           |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `cookie[].groupId` / `cookie[].commonGroup` | The group this cookie belongs to. Optional. Not needed when the cookie is defined in an `optionalCookies` / `requiredCookies.group` If `commonGroup` is used and the group itself is not listed, it is auto-added from `content.json`. |
+| `cookie[].commonCookie`                     | Optional. Use when the cookie is common.                                                                                                                                                                                     |
+| `cookie[].id`                               | The ID of the consent. This ID is saved in the consent cookie. **Must be unique.** Only for custom cookies.  Cannot be set for a common cookie.                                                                                         |
+| `cookie[].name`                             | Readable cookie name. Only for custom cookies.  Cannot be set for a common cookie.                                                                                                                                                      |
+| `cookie[].hostName`                         | Hostname where it is used. Only for custom cookies.  Cannot be set for a common cookie.                                                                                                                                                |
+| `cookie[].description`                      | Cookie usage description. Only for custom cookies.  Cannot be set for a common cookie.                                                                                                                                                  |
+| `cookie[].expiration`                       | When does the cookie expire. Only for custom cookies. Cannot be set for a common cookie.                                                                                                                                                |
+[Table 4:Cookie information properties]|
+
+##### Callbacks
+The `content` object passed to the component can also include callback functions:
+
+| Property             | Usage                                                                                                 | Arguments                                                                                                                                                                                     |
```

Comment:
Should we add column `usage`, that would describe at least one scenario where the user would wish to use the callback?